controller/controller.go

```diff
@@ -51,6 +53,9 @@ const (
 
 	// DefaultServiceThreads Default number of service worker threads to start with the controller
 	DefaultServiceThreads = 10
+
+	// DefaultIngressThreads Default number of ingress worker threads to start with the controller
```

Comment:
```suggestion
	// DefaultIngressThreads is the default number of ingress worker threads to start with the controller
```

otherwise it will look weird in Godoc